### DISCUSSION TITLE
Multi Cluster Management

### DISCUSSION THREAD

**Discussion by jcamu** [fluxcd/flux2#2892] [Q&A] [answered]
Hello,

In a multi cluster management (>20) context what would be the pros and cons of one centralized manager cluster (managing remote clusters) vs installing flux on every cluster for own management  ?

Thanks.

Regards.

**Comment by stefanprodan** [comment, marked as answer]
The only con I see when using Flux on the management cluster only, is the fact that the management cluster becomes a SPOF for your whole fleet. If that cluster goes down, or if the Flux upgrade fails, then all your clusters will not be synced with their designed state. Running a Flux per cluster, makes your CD solution more resilient to failures and it also gives you the opportunity to tests a Flux upgrade on some of the clusters instead of all or nothing.
